### ISSUE TITLE
[Feature Request] List Organizations in resource group / subscription

### ISSUE THREAD

**Issue by rvf148** [Azure/azure-devops-cli-extension#611]
I cannot list Azure DevOps organizations.

I would like to list Azure DevOps organizations per:
- resource group
- subscription
- tenant

This way I can create automated loops over Azure DevOps organzation resources.

**Comment by geverghe** [comment]
@rvf148 - Apologies for the delayed response. I have been doing some digging and here are my learning:
1. Azure DevOps organization is not related to an Azure Subscription. Further, there is no resource allocation that occurs on creation of an organization for it be distributed across a resource group/tenant.
2. Further, you could still create an Azure DevOps organization without an Azure subscription.

Therefore, could you clarify intent here?

**Comment by rvf148** [comment]
@geverghe In companies with a large amount of Azure DevOps organisations, there is a need to automate certain tasks, for example adjusting service endpoints in all projects of all Azure Devops organisations. In this case I would like to be able to iterate through all organisations I have access for, instead of adjusting manually the list op Azure DevOps organisations. Another scenario could be searching for the usage of certain features across Azure DevOps organisations.

**Comment by skeeler** [comment]
Agree, it would be great to have an "az devops organization" to iterate over all the organizations that I am a member/user in. For example, I can see such a list at the URL: https://app.vsaex.visualstudio.com/me. As @rvf148  mentioned, for large organizations with several (possibly many) organizations, it would be nice for scripting to handle this dynamically rather than statically.

**Comment by Daleyon** [comment]
As an alternative, I use a combination of Azure (Logic App, SQL, Pipelines) to maintain a list of all orgs for my employer.
The Azure Logic App uses the [Azure DevOps connector List Organizations ](https://learn.microsoft.com/en-us/connectors/visualstudioteamservices/#list-organizations) on recurrence.

However, much time and money would be saved if we had something like 'az devops orgs' or 'az devops accounts'! 😉
